src/NuGet.Clients/NuGet.PackageManagement.UI/Models/Package/VulnerableCapabilityBase.cs

```diff
@@ -40,7 +40,7 @@ public PackageVulnerabilitySeverity VulnerabilityMaxSeverity
             {
                 if (!IsVulnerable || Vulnerabilities is null)
                 {
-                    throw new InvalidOperationException("Vulnerabilities is empty");
+                    return PackageVulnerabilitySeverity.Unknown;
```

Comment:
Should we also check for Vulnerabilities.Count == 0?

Reply:
This is handled by IsVulnerable already.